.github/workflows/linting.yml

```diff
@@ -0,0 +1,61 @@
+name: Lints
+
+on:
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  store-pr:
```

Comment:
don't think the automatic linting issue creation is necessary, seems unnecessarily complicated, and the point of the lint/formatting checks is to avoid them from entering `main` anyways.